src/test/java/hello/HelloWorldTest.java

```diff
@@ -0,0 +1,39 @@
+package hello;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import com.bankofspring.hello.GreetingService;
+
+public class HelloWorldTest {
```

Comment:
I thought that if I did that I'd have to choose the context file as part of the runner. Maybe you can explain why this is a good thing at the meeting?